### ISSUE TITLE
Overlay cut off after third entry

### ISSUE THREAD

**Issue by hir0xygen** [trigg/Discover#336]
### **Steps to Reproduce**
1. Start the overlay

### **Expected behavior**
The full list of users can be displayed.

### **Problematic behavior**
It cuts off after the fourth user.

### **Desktop (please complete the following information):**
 - Operating System / Distribution : Artix (dinit)
 - Desktop Environment : Cinnamon 6.0.4

### **Installation Method**
 - AUR (discover-overlay-git)

### **Discord client information**
 - Install method : pacman
 - Version : stable 300109 (ce7e96b) Host 0.0.55 x64 Build Override: N/A Linux 64-bit (6.9.2-zen1-1-zen)

### **Additional context**
![2024-06-10_19-53](https://github.com/trigg/Discover/assets/150862116/63619c80-f08d-476e-b80a-32874f9f6833)

**Comment by trigg** [comment]
This is a mix of problems. TLDR fix at bottom

Firstly, you're using Cinnamon, which has a known bug which we work around forcibly by ensuring voice chat is set to `floating` style.

Because you're using floating style and haven't set the window position it uses default, which is quite small.

Default `overflow` option is `None` which just cuts off at the limit

## Fix 
Suffice it to say, you should go to `discover overlay configure` -> `Voice` -> `Place Window`  and resize & position the window. Beware that it shouldn't overlap panels as that causes unexplained errors in cinnamon. Linked to #98

**Comment by hir0xygen** [comment]
Thanks!

Wasn't aware of the Cinnamon issue.

I don't have an option called "Place Window", though?
Sorry in advance if I'm stupid.
![image](https://github.com/trigg/Discover/assets/150862116/645e2675-088f-4b8b-b52b-5fc934c1295a)

**Comment by trigg** [comment]
No, you're right. It should be below the grayed out 'Anchor to edge'... That's a concern for me then!

**Comment by trigg** [comment]
I've added a commit which should correct this, after updating to it and going there,  you should see 'Place window' button below the grayed out option. Let me know if there are other issues

**Comment by hir0xygen** [comment]
Working well now, thanks!

**Comment by peterweissbier** [comment]
i am unsure on how to "pull this commit" so i can see the added place window button. i just reinstalled the program via the aurs git version and the button still wont appear for me, i tried updating with git pull but it says its already the latest version. what am i missing here please?
